group_vars/location_gubener/snmp.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+snmp_devices:
+  - hostname: gubener-sama-5ghz
```

Comment:
The current hostname is `gubener-sama`, according to assignment ¹ and monitoring ².


[1] https://github.com/Freifunk-Spalter/bbb-configs/blob/81429c151f192651867b2cfc2f75153b1f40a6fb/group_vars/location_gubener/networks.yml#L30

[2] http://monitor.berlin.freifunk.net/host.php?h=sama-sued-5ghz&p=snmp